### DISCUSSION TITLE
What is "dev" field in lock file for?

### DISCUSSION THREAD

**Discussion by Olyno** [pnpm/pnpm#5292] [Q&A] [answered]
Hi 👋🏻 

I was wondering, but what does the "dev" field in the lock file correspond to, please?

![See image here: https://cdn.discordapp.com/attachments/1014875162597929082/1014887900728545280/unknown.png](https://cdn.discordapp.com/attachments/1014875162597929082/1014887900728545280/unknown.png)

**Comment by zkochan** [comment, marked as answer]
`dev: true` means the dependency is only used by dev dependencies.

**Reply by Olyno** [reply]
Ah okay, thank you! Is there a reason why there is this change recently in the lock file?

**Reply by zkochan** [reply]
maybe you moved some dependency from prod deps to dev deps.

**Reply by Olyno** [reply]
Alright, thank you!
